flags.bzl

```diff
@@ -310,3 +310,87 @@ FLAGS = {
         """,
     ),
 }
+
+# Flags being flipped in the next Major release
+# Based on https://github.com/bazelbuild/bazel/issues?q=is%3Aissue%20state%3Aopen%20label%3Amigration-ready
+# buildifier: keep-sorted
+MIGRATIONS = {
```

Comment:
The alternative to this would be to generate all of them under a `migrate` config.

Either way, when would a user apply this? As an alternative to bazelisk's `--strict` and `--migrate` options or as an additional feature to protect the incompatible flags once they work?

Reply:
I think it's not desirable that the user has to think about whether `--config=migrate` belongs on this particular invocation of Bazel. Either the repo is updated to be clean with these flags, or it isn't - that state should just be represented as a constant in bazelrc.

I imagine the sequence is
1. run bazelisk with `--migrate` to turn on the flags individually
2. set `strict=True` in the bazelrc preset rather than passing `--strict` to bazelisk